src/index.html

```diff
@@ -11,7 +11,194 @@
+            required
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date-of-birth"
+            required
+          >
+        </label>
+
+        <div>
+          <label
+            class="form-label"
+            for="terms-and-conditions"
+          >
+            I accept the terms of the agreement
+          </label>
+          <input
+            id="terms-and-conditions"
+            type="checkbox"
+            name="terms_and_conditions"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Registration
+        </legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          An interesting fact about you?
+        </legend>
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="y"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="n"
+            >
+            No
+          </label>
+
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go bed?
+          <input
+            type="time"
+            name="time"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="favorite_brand"
+            required
+            multiple
+            size="4"
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label>
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Additional information:
+        </legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            autocomplete="off"
          ></textarea>
```

Reply:
to avoid this
![image](https://user-images.githubusercontent.com/98764641/210974596-1b7ff0c6-22ce-4598-b83d-a3d680f9f641.png)